gcloud-java-logging/src/main/java/com/google/gcloud/logging/spi/v2/LoggingServiceV2Settings.java

```diff
@@ -369,4 +213,178 @@ public Object extractNextToken(ListMonitoredResourceDescriptorsResponse payload)
               return payload.getResourceDescriptorsList();
             }
           };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'deleteLogSettings' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2388714616/issues/source?bid=3123104&fileBranchId=3228843#l220)